android/java/res/layout/new_tab_page_incognito.xml

```diff
@@ -43,7 +43,7 @@
                 app:layout_constraintGuide_end="32dp" />
 
             <!-- Keep the ID below as it's been tested in Chromium code by IncognitoNewTabPageStation. -->
-            <ImageView
+            <!-- <ImageView
```

Comment:
what if we set  a property here hiding these views. we won't have to comment the whole block of code then

Reply:
i will have to set the visibility none, ht, wd , padding, margin = 0. to keep the ui aligned